### PR TITLE
Create docs aug2023

### PR DESCRIPTION
**Pull request by DBees**

### CODE REVIEW

File: docs/doc-src/software-description.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+  Copyright (c) 2023 OpenHW Group
```
Comment (MikeOpenHWGroup) [inline docs/doc-src/software-description.rst:1]: Dito.

**Review by MikeOpenHWGroup — CHANGES_REQUESTED** [review]
A few formatting comments.  Also, you'll need a `readthedocs.yaml` at the top-level if you plan to use RTD to publish these docs on the web with RTD.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Documentation
+
+## Prerequisites
+
+You need to install the following Sphinx extensions (in addition to the
+requirements in `../python-requirements.txt`).
```
[the remark this answers is not in the document]
Comment (DBees) [reply docs/README.md:6]: guilty

**Comment by DBees** [comment]
@MikeOpenHWGroup thanks for the comments. I've corrected these and a few other things also.

**Comment by MikeOpenHWGroup** [comment]
> @MikeOpenHWGroup thanks for the comments. I've corrected these and a few other things also.

Yep, saw that.  The updates look good.  A couple things still outstanding:
1. The path to the requirements file in `docs/README.md` needs to be fixed.
2. We should also add the `build.os` statements to the `.readthedocs.yaml` file (in your defence, we heard about this from RTD _after_ you opened this PR).

**Comment by DBees** [comment]
@MikeOpenHWGroup should be clean now

**Pull request merged**